### DISCUSSION TITLE
Unable to upload points

### DISCUSSION THREAD

**Discussion by ThomasDV1984** [opengisch/QField#3600] [Q&A]
I have an inventory with a point layer and some tables with no geometry in a 1:n relation. I can collect point but when pushed back only the chances in the non geometry tables are applied. The points layer gives an error in the chancelog stating: No layer with ID"". 
![image](https://user-images.githubusercontent.com/77888654/200667912-7ccfa829-a389-4284-ab35-897956295984.png)
Anyone know with could be the issue or if i can get the points in my project?
![image](https://user-images.githubusercontent.com/77888654/200668251-d999d30a-7057-4896-9143-a4da58bfa5f8.png)
![image](https://user-images.githubusercontent.com/77888654/200668375-45451806-02fb-4288-bdae-1201bc33c2f5.png)

**Comment by suricactus** [comment]
What QGIS/QFieldSync version did you use for this project?

**Reply by ThomasDV1984** [reply]
![image](https://user-images.githubusercontent.com/77888654/200670724-7e863a6f-f7b4-4b3a-b2e7-1a9bf23fac82.png)
3.26.2

**Reply by ThomasDV1984** [reply]
Is this problem solved? Or what action should I take? And will I be aible to get the data or is it lost??
